File: code/8f116.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() 
{
    int a,b,c;
    cin>>a>>b>>c;
    cout<<min(a,min(c,b));
    cout<<" "<<max(a,max(b,c));
    return 0;
}

File: code/8f116.pas
program Bai116;
uses math;
var SNN, SLN, a, b, c: integer;
begin
  readln(a, b, c);
  SNN := min(a, min(b,c));
  SLN := max(a, max(b,c));
  writeln(SNN, '  ', SLN);
end.